samples/nrf9160/modem_shell/README.rst

```diff
@@ -734,6 +734,69 @@ After programming the application and all prerequisites to your development kit,
 
 #. Type any of the commands listed in the Overview section to the terminal. When you type only the command, the terminal shows the usage, for example ``sock``.
 
+Getting nRF9160 DK out-of-the-box and to nRF Cloud
+==================================================
+
+To program the certificates and connect to nRF Cloud, complete the following steps:
+
+1. `Download nRF Connect for Desktop`_.
+#. Update the modem firmware on the on-board modem of the nRF9160 DK to the latest version as instructed in :ref:`nrf9160_gs_updating_fw_modem`.
+#. Build and program the MoSh to the nRF9160 DK using the default MoSh configuration (with REST as a transport):
+
+.. code-block:: console
+
+   $ west build -p -b nrf9160dk_nrf9160_ns -d build
+   $ west flash -d build
+
+#. Get certificates from nRF Cloud as explained in :ref:`downloading_cloud_certificate`.
+#. In the MoSH terminal, power off the modem and start the AT command mode:
+
+   .. code-block:: console
+
+      mosh:~$ link funmode -0
+      mosh:~$ at at_cmd_mode start
+
+#. Disconnect the MoSh terminal.
+#. Connect and use `LTE Link Monitor`_ to store the certificates to the modem (default security tag).
+   See `Managing credentials`_ in the LTE Link Monitor user guide for instructions.
+#. Reconnect the MoSh terminal and press ``ctrl-x`` and ``ctrl-q`` to exit the AT command mode.
+#. Set the modem to normal mode to activate LTE.
+   Observe that LTE is getting connected.
+
+   .. code-block:: console
+
+      mosh:~$ link funmode -1
+
+#. Perform just-in-time provisioning (JITP) with nRF Cloud through REST:
+
+   .. code-block:: console
+
+      mosh:~$ cloud_rest jitp
+
+   You only need to do this once for each device.
+
+#. Follow the instructions for JITP printed in the MoSh terminal.
+#. To complete the user association in nRF Cloud side, navigate to `nRF Cloud`_. Click the large plus sign in the upper-left corner. In the :guilabel:`Device ID` field, enter the device id that was instructed by MoSh.
+   When device has been added the message :guilabel:`Device added to account. Waiting for it to connect...` appears.
+   When the message has disappeared, go to the menu on the left and click :guilabel:`Devices`. You can now see your MoSh device in a list.
+#. Send MoSh device information to nRF Cloud.
+
+   It might take a while for the data to appear in the nRF Cloud UI.
+
+   .. code-block:: console
+
+      mosh:~$ cloud_rest shadow_update
```

Comment:
```suggestion
#. Send MoSh device information to nRF Cloud:

   .. code-block:: console

      mosh:~$ cloud_rest shadow_update

   It might take a while for the data to appear in the nRF Cloud UI.
```

Reply:
changed